docs/features/opencdc-record.mdx

```diff
@@ -70,6 +70,23 @@ expand its functionality. Notice that all these fields use a dot notation syntax
 to indicate what they refer to, preventing accidental clashes. Here are the ones
 you can find:
 
+### `opencdc.version`
+
+Contains the version of the OpenCDC format (e.g., "v1"). This field exists to
+ensure the OpenCDC format version can be easily identified in case the record
+gets marshaled into a different untyped format (e.g. JSON).
+
+```json5
```

Comment:
This `json5` thingy doesn't seem to work for me, while the `json` directive used above works. Maybe rename `json5` to `json` for nicer code highlighting?